src/edge_sequences.jl

```diff
@@ -44,3 +44,8 @@ end
 @traitfn function edge_sequence(::Algorithm"parallel", g::::(!IsDirected))
   return [[e] for e in vcat(edges(g), reverse.(edges(g)))]
 end
+
+function post_order_dfs_edges_region(g::AbstractGraph, region)
+  es = post_order_dfs_edges(g, first(region))
+  return filter(e -> !((src(e) ∈ region) && (dst(e) ∈ region)), es)
+end
```

Comment:
So if I understand correctly, this returns the edges directed towards the region, excluding any edges in the region?

Reply:
Yes exactly, which is exactly the sequence of edges one will need to orthogonalize along to make `region` the `ortho_region` of a `ttn`. 
Maybe we could be a bit tighter with the logic here and demand `region` is a contiguous group of vertices. I suppose it should also probably end up being a `NamedGraphs` function at some point.

Reply:
Change the logic to fill in the region if it is not contiguous